src/constants/api.ts

```
@@ -54,6 +54,8 @@ export const END_POINTS = {
 		`/app/teamspaces/${teamspaceId}/users/${userId}/chat-channels/status`,
 	GET_UNREAD_MESSAGE_COUNT: (teamspaceId: number) =>
 		`/teamspaces/${teamspaceId}/unread-count`,
+	POST_TEAMSPACE_ROLE: (teamspaceId: number) =>
+		`teamspaces/${teamspaceId}/tags`,
```

ESLint

[error] Replace `⏎↹↹` with `·`

(prettier/prettier)